source/components/molecules/EditableList/EditableList.js

```diff
@@ -200,18 +211,14 @@ function EditableList({
       )}
     >
       <EditableListBody>
-        {inputs.map((input, index) => (
+        {inputs.map((input, index) => [
           <EditableListItem
             colorSchema={colorSchema}
             editable={editable}
             key={input.key}
             error={error ? error[input.key] : undefined}
             activeOpacity={1.0}
-            onPress={() => {
-              if (editable && inputRefs.current?.[index]?.focus) inputRefs.current[index].focus();
-              else if (editable && inputRefs.current?.[index]?.togglePicker)
-                inputRefs.current[index].togglePicker();
-            }}
+            onPress={(index) => handleListItemPress(index)}
```

Comment:
Correct me if i'm wrong but the index defined here is the onPress event, should it not be defined as:
```
onPress={() => handleListItemPress(index)}

// the current code is rather a definition of:
onPress={(event) => handleListItemPress(event)}
```